js/app.js

```diff
@@ -0,0 +1,367 @@
+		//3. A list of students could be modeled with an array
+	//B. Easy does it
+		//1. Create an array that contains three quotes and store it in a variable called quotes.
+		const quotes = ["To infinity and beyond","The first rule is you do not talk about fight club","All your base are belong to us"];
+	//C. Accessing elements
+		const randomThings = [1, 10, "Hello", true];
+		//1.	
+		randomThings[0];
+		//2.
+		randomThings[2] = "World";
+		//3.
+		console.log(randomThings);
+	//D. Change values
+		const ourClass = ["Salty", "Zoom", "Sardine", "Slack", "Github"]
+		//1. 
+		ourClass[2];
+		//2.
+		ourClass[4] = "Octocat";
+		//3.
+		ourClass.push("Cloud City");
+	//E. Mix It Up
+		const myArray = [5, 10, 500, 20]
+		//1.
+		myArray.push("Egon");
+		myArray.push("Bolas");
+		//2.
+		myArray.splice(0,1);
+		//3.
+		myArray.splice(0,0,"Bob Marley");
+		//4.
+		myArray.pop();
+		//5.
+		myArray.reverse();
+	//F. Biggie Smalls
+		const biggieSmalls = 144;
+		if(biggieSmalls < 144) {
+			console.log("little number");
+		} else if(biggieSmalls >= 144) {
+			console.log("big number");
+		}
+	//G. Monkey in the Middle
+		const monkey = 8;
+		if(monkey < 5) {
+			console.log("little number");
+		} else if(monkey > 10) {
+			console.log("big number");
+		} else {
+			console.log("monkey");
+		}
+	//H. What's in Your Closet?
+		const kristynsCloset = [
+		  "left shoe",
+		  "cowboy boots",
+		  "right sock",
+		  "GA hoodie",
+		  "green pants",
+		  "yellow knit hat",
+		  "marshmallow peeps"
+		];
+
+		// Thom's closet is more complicated. Check out this nested data structure!!
+		const thomsCloset = [
+		  [
+		    // These are Thom's shirts
+		    "grey button-up",
+		    "dark grey button-up",
+		    "light blue button-up",
+		    "blue button-up",
+		  ],[
+		    // These are Thom's pants
+		    "grey jeans",
+		    "jeans",
+		    "PJs"
+		  ],[
+		    // Thom's accessories
+		    "wool mittens",
+		    "wool scarf",
+		    "raybans"
+		  ]
+		];
+		//1.
+		console.log("Kristyn is rocking that " + kristynsCloset[2] + " today!");
+		//2.
+		kristynsCloset.splice(6,0,"raybans");
+		//3. 
+		kristynsCloset[5] = "stained knit hat";
+		//4.
+		console.log(thomsCloset[0][0]);
+		//5.
+		console.log(thomsCloset[1][1]);
+		//6.
+		console.log(thomsCloset[2][2]);
+		//7.
+		console.log("Thom is looking fierce in a " + thomsCloset[0][0] + ", " + thomsCloset[1][1] + " and " + thomsCloset[2][2] + "!");
+		//8.
+		thomsCloset[1][2] = "Footie Pajamas";
+//IV. Functions
+	//A. printGreeting
+	//skipped
+	//B. printCool
+	const printCool = (name) => {
+		return name + " is cool";
+	}
+	console.log(printCool("Jeff"));
+	//C. calculateCube
+	const calculateCube = (number) => {
+		return number**3;
+	}
+	console.log(calculateCube(4));
+	//D. isVowel
+	const isVowel = (char) => {
+		if(char === "a" || char === "A" || char === "e" || char === "E" || char === "i" || char === "I" || char === "o" || char === "O" || char === "u" || char === "U") {
```

Comment:
Think of a way to cut down the number of comparisons...

For example, at the least, you could `.toLowerCase()` char before comparing, and get rid of half of 'em